src/constants.ts

```diff
@@ -2,27 +2,88 @@ import { Endpoints } from './models/core/Endpoints';
 
 export const defaultApiVersion = 20220511;
 
-export const defaultEndpoints: Required<Endpoints> = {
-  universalSearch: 'https://liveapi.yext.com/v2/accounts/me/search/query',
-  verticalSearch: 'https://liveapi.yext.com/v2/accounts/me/search/vertical/query',
-  questionSubmission: 'https://liveapi.yext.com/v2/accounts/me/createQuestion',
-  status: 'https://answersstatus.pagescdn.com',
-  universalAutocomplete: 'https://liveapi-cached.yext.com/v2/accounts/me/search/autocomplete',
-  verticalAutocomplete: 'https://liveapi-cached.yext.com/v2/accounts/me/search/vertical/autocomplete',
-  filterSearch: 'https://liveapi-cached.yext.com/v2/accounts/me/search/filtersearch',
-} as const;
+/**
+ * Defines the cloud region of the API domains.
+ *
+ * @public
+ */
+export enum CloudRegion {
+  US = 'us',
+  EU = 'eu',
+}
+
+/**
+ * Defines the environment of the API domains.
+ *
+ * @public
+ */
+export enum Environment {
+  PROD = 'prod',
+  SANDBOX = 'sbx',
+}
+
+/**
+ * Provides methods for getting various endpoints.
+ *
+ * @internal
+ */
+export class EndpointsProvider {
```

Comment:
nit: I think this would fit better outside of a constants file. Maybe we could make a `provideEndpoints` file, and move this class and that function there as well? I think `provideEndpoints.ts` would fit in either src or src/utils

Reply:
Renamed the file 👍

Reply:
Renamed the file 👍